Sources/GeoSwift.swift

```diff
@@ -75,3 +75,4 @@ public extension GeoCoordinate2D {
     }
 }
 
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/petrpavlik/GeoSwift/blob/39f3c59b1843ddba396fdb7110d365a9624987f6/Sources/GeoSwift.swift#L78"></span>Files should have a single trailing newline.</td>
    </tr>
  </tbody>
</table>